### PR TITLE
154440133 business-id report

### PR DESCRIPTION
**Pull request by rckopteri**
# Added
 - Business id report to admin panel

### CODE REVIEW

File: ote/src/clj/ote/services/admin.clj [hunk 1]
```diff
@@ -73,6 +76,36 @@
          search-params
          {:specql.core/order-by ::t-service/operator-name})))
 
+(defn distinct-by [f coll]
+  (let [groups (group-by f coll)]
+    (map #(first (groups %)) (distinct (map f coll)))))
+
+(defn- business-id-report [db user query]
+  (let [services (if
```
Comment (tatut) [inline ote/src/clj/ote/services/admin.clj]: Use `when` instead of `(if condition value nil)` pattern

File: ote/src/clj/ote/services/admin.clj [hunk 2]
```diff
@@ -73,6 +76,36 @@
...
+                   nil)
+        operators (if
+                   (or
+                     (nil? (:business-id-filter query))
+                     (= :ALL (:business-id-filter query))
+                     (= :operators (:business-id-filter query)))
+                    (fetch-operator-business-ids db)
+                   nil)
+
+        report (cond
```
Comment (tatut) [inline ote/src/clj/ote/services/admin.clj]: Why is this  `cond` structure necessary? just `(concat services operators)`

File: ote/src/cljs/ote/views/admin.cljs [hunk 3]
```diff
@@ -49,6 +49,55 @@
...
+              [ui/table-header-column "Sähköposti"]]]
+          [ui/table-body {:display-row-checkbox false}
+           (doall
+             (for [{:keys [operator business-id phone email]} results]
+               [ui/table-row {:selectable false}
+                [ui/table-row-column operator]
+                [ui/table-row-column business-id]
+                [ui/table-row-column phone]
+                [ui/table-row-column email]
+                ]
```
Comment (tatut) [inline ote/src/cljs/ote/views/admin.cljs]: Closing parens

File: ote/src/clj/ote/services/admin.clj (hunk 1 above)
Comment (rckopteri) [reply ote/src/clj/ote/services/admin.clj]: Changed

File: ote/src/clj/ote/services/admin.clj (hunk 2 above)
Comment (rckopteri) [reply ote/src/clj/ote/services/admin.clj]: Removed cond and replaced it with concat

File: ote/src/cljs/ote/views/admin.cljs (hunk 3 above)
Comment (rckopteri) [reply ote/src/cljs/ote/views/admin.cljs]: parens fixed

**Pull request merged**